src/utils/helpers/sanitizeInput.js

```diff
@@ -0,0 +1,5 @@
+function sanitizeInput(input) {
```

Comment:
I would merge createDate.js and sanitizeInput.js into one file. named it utilities.js